libcsight/source/csight/RaidPokemon.cpp

```diff
@@ -66,6 +66,9 @@ namespace csight::raid {
 
   bool RaidPokemon::GetIsShiny() { return m_shineType > shiny::None; }
 
+  // TODO: find how to obtain raid pokemon Gigantamax status
+  bool RaidPokemon::GetIsGigantamax() { return false; }
```

Comment:
This comes from the raid templates, so they'll need to be added later.  I'm not opposed to that being added outside of this PR though.

Ideally the templates are read from the game instead of being hardcoded into csight, but I've been a bit lazy and haven't looked for them.